packages/components/src/toggle-group-control/toggle-group-control-option-base/component.tsx

```diff
@@ -79,7 +79,12 @@ function ToggleGroupControlOptionBase(
 	);
 
 	return (
-		<LabelView className={ labelViewClasses } data-active={ isActive }>
+		<LabelView
+			className={ labelViewClasses }
+			data-active={ isActive }
+			// Necessary for the ToggleGroupControlBackdrop component to render properly
+			data-toggle-group-control-option-wrapper="true"
```

Comment:
The `data-toggle-group-control-option-wrapper` is used as a way for the backdrop component to always find the correct `ToggleGroupControl`'s "option wrapper", which is used to compute the correct position and size to apply to the backdrop.